homeassistant/components/environment_canada/sensor.py

```diff
@@ -249,6 +260,8 @@ def native_value(self):
         sensor_type = self.entity_description.key
         if sensor_type == "timestamp":
             return self._ec_data.metadata.get("timestamp")
+        if sensor_type == "aqhi":
+            return self._ec_data.current
```

Comment:
Since you are using entity descriptions, you could extend that with a `value_fn` containing a lambda method. That would remove the need to figure out what to use in this method.

Reply:
Fixed. Thanks!